changes/805e1b53761ec5c94f585b65dbd0f5b4.yaml

```
@@ -0,0 +1,5 @@
+---
+desc: Fixed a bug where growing a slab when updating a value would cause `scanByFull()` to iterate the same key twice.
```

Not sure how to word this or that the details will be too relevant to anyone but maybe something like this?
```suggestion
desc: Fixed a bug where operations which caused a ``Scan.bump()`` on a non-dupsort DB could cause the Scan to yield the same key twice.
```

---

Yea, this should def get super simplified. Something like

Fixed a bug in storage iteration which could cause an item to be yielded twice if a very narrow race window was encountered.